enhancements/storage/manila-csi-driver-operator.md

```diff
@@ -142,6 +142,16 @@ Actions:
 
 The operator will run containerized and End-to-End tests and also push the resulting image in the OCP Quay account.
 
+#### Implement Secondary NIC support
+
+By default in OSP Manila uses additional network for volumes. This means that in addition to the main network, we must connect worker nodes to the appropriate storage network to be able to successfully mount the volumes.
+
+Actions:
+
+- Add the ability to specify additional networks and security groups in the installer configuration.
```

Comment:
Is the installer team aware?

Reply:
Yes! @abhinavdahiya knows about this requirement. https://github.com/openshift/installer/pull/3291

Reply:
can you include the expected design of these fields are going to be added here, from https://github.com/openshift/installer/pull/3291

and would like to see what validations etc. will be added to these new fields.

Reply:
Done!

Reply:
And regarding to validations: https://github.com/openshift/installer/pull/3291/commits/cf318fc3536ea51cd8832b495498d1eb8b1f82c4